vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -934,27 +943,27 @@ window.Vaadin.Flow.gridConnector = {
 
     grid.addEventListener('grid-dragstart', e => {
 
-        if (grid._isSelected(e.detail.draggedItems[0])) {
-            // Dragging selected (possibly multiple) items
-            if (grid.__selectionDragData) {
-                Object.keys(grid.__selectionDragData).forEach(type => {
-                    e.detail.setDragData(type, grid.__selectionDragData[type]);
-                });
-            } else {
-                (grid.__dragDataTypes || []).forEach(type => {
-                    e.detail.setDragData(type, e.detail.draggedItems.map(item => item.dragData[type]).join('\n'));
-                });
-            }
+      if (grid._isSelected(e.detail.draggedItems[0])) {
+      // Dragging selected (possibly multiple) items
+      if (grid.__selectionDragData) {
```

Comment:
Lots of strange indentation changes in this file. Rather only include the relevant changes.